src/discussions/post-comments/comments/comment/CommentEditor.jsx

```diff
@@ -97,6 +111,34 @@ const CommentEditor = ({
     }
   }, [formRef]);
 
+  useEffect(() => {
+    const draftHtml = draftContent.getDraftContent(parentId, threadId, id) || rawBody;
+    setEditorContent(draftHtml);
+    initialValues.comment = draftHtml;
```

Comment:
`setEditorContent` will cause a re-rendering and a comment value will get assigned.